src/main/resources/assets/bloodmagic/lang/fr_FR.lang

```diff
@@ -1,239 +1,250 @@
+tile.BloodMagic.rune.selfsacrifice.name=Rune de l'Autosacrifice
+tile.BloodMagic.rune.displacement.name=Rune de Dislocation
+tile.BloodMagic.rune.capacity.name=Rune de Capacité
+tile.BloodMagic.rune.augcapacity.name=Rune de Capacité Supérieur
+tile.BloodMagic.rune.orb.name=Rune de l'Orbe
+tile.BloodMagic.rune.acceleration.name=Rune d'Accélération
+tile.BloodMagic.rune.charging.name=Rune de Chargement
+
+
+tile.BloodMagic.ritualStone.blank.name=Pierre Rituelle
+tile.BloodMagic.ritualStone.water.name=Pierre Rituelle d'Eau
+tile.BloodMagic.ritualStone.fire.name=Pierre Rituelle du Feu
+tile.BloodMagic.ritualStone.earth.name=Pierre Rituelle de la Terre
+tile.BloodMagic.ritualStone.air.name=Pierre Rituelle de l'Air
+tile.BloodMagic.ritualStone.dusk.name=Pierre Rituelle du Crépuscule
+tile.BloodMagic.ritualStone.dawn.name=Pierre Rituelle de l'Aube
+
+tile.BloodMagic.bloodstonebrick.large.name=Brique Large de Roche-Sang
+tile.BloodMagic.bloodstonebrick.brick.name=Brique de Roche-Sang
+tile.BloodMagic.crystal.large.name=Groupe de Cristaux
+tile.BloodMagic.crystal.brick.name=Bloc de Groupe de Cristaux
+tile.BloodMagic.bloodLight.name=Lumière Sanguine
+tile.BloodMagic.spectralBlock.name=Bloc Spectral
+tile.BloodMagic.phantom.name=Bloc Fantôme
+tile.BloodMagic.incenseAltar.name=Autel Encens
+
+tile.BloodMagic.teleposer.name=Téléposeur
+tile.BloodMagic.soulForge.name=Hellfire Forge
+tile.BloodMagic.alchemyTable.name=Table d'Alchimie
+tile.BloodMagic.demonCrucible.name=Creuset Diabolique
+tile.BloodMagic.demonPylon.name=Pylône Diabolique
+tile.BloodMagic.demonCrystallizer.name=Demon Crystallizer
+
+tile.BloodMagic.masterRouting.name=Master Routing Node
+tile.BloodMagic.outputRouting.name=Output Routing Node
+tile.BloodMagic.inputRouting.name=Input Routing Node
+tile.BloodMagic.itemRouting.name=Routing Node
+
+tile.BloodMagic.path.wood.name=Chemin en Bois
+tile.BloodMagic.path.woodtile.name=Chemin en Bois Carrelé
+tile.BloodMagic.path.stone.name=Chemin en Pierre
+tile.BloodMagic.path.stonetile.name=Chemin en Pierre Carrelée
+tile.BloodMagic.path.wornstone.name=Chemin en Pierre Usé
+tile.BloodMagic.path.wornstonetile.name=Chemin en Pierre Usé Carrelée
+tile.BloodMagic.path.obsidian.name=Chemin en Obsidienne
+tile.BloodMagic.path.obsidiantile.name=Chemin en Obsidienne Carrelée
+
+tile.BloodMagic.dimensionalPortal.name=Portail Dimensionnel
+tile.BloodMagic.bloodTank.name=Réservoir de Sang
+
+tile.BloodMagic.demonCrystalDEFAULT.name=Groupe de Cristaux d'une Volonté Diabolique
+tile.BloodMagic.demonCrystalCORROSIVE.name=Groupe de Cristaux d'une Volonté Corrosive
+tile.BloodMagic.demonCrystalDESTRUCTIVE.name=Groupe de Cristaux d'une Volonté Destructive
+tile.BloodMagic.demonCrystalVENGEFUL.name=Groupe de Cristaux d'une Volonté Vengeuse
+tile.BloodMagic.demonCrystalSTEADFAST.name=Groupe de Cristaux d'une Volonté Inébranlable
+
+# Fluids
+fluid.lifeEssence=Essence de Vie
 
 #Tooltips
 tooltip.BloodMagic.extraInfo=&9-Tenir shift pour plus d'info-
```

Comment:
«info» is only a part of the word, but usely understood. the correct word is : «informations»